web planner

```diff
@@ -0,0 +1,70 @@
+Notebook page
+Email page
+news page
+advertisements page
+Publish page
+communication page
```

Comment:
Anybdy is welcome to communicate with us and feel happy to speak out of their minds